snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -145,7 +145,7 @@ open class Snapshot: NSObject {
                 print("Can't prepare environment. Simulator home location is inaccessible. Does \(simulatorHostHome) exist?")
                 return nil
             }
-            homeDir = homeDirUrl
+            homeDir = URL(fileURLWithPath: homeDirUrl.path)
```

Comment:
Can we change line 144 to say `guard let homeDirUrl = URL(fileURLWithPath: simulatorHostHome)...` instead of making a URL here with another URL's path?